core/spec.md

```diff
@@ -1055,6 +1059,25 @@ by the implementation of the Registry Service. This information will usually
 be used by tooling that does not have advance knowledge of the type of data
 stored within the Registry.
 
+When a new implementation is first created, or a new specifcation is based
+on the xRegistry specification, then it MAY choose to modify the core
+attributes to fit their needs. The changes allowed are restricted to the
```

Comment:
their => its? (Given the "it may choose")

Reply:
done